analytics/filesystem/file_module_test.go

```diff
@@ -64,6 +73,7 @@ func TestFileLogger_LogObjects(t *testing.T) {
 	defer os.RemoveAll(TEST_DIR)
 	if fl, err := NewFileLogger(TEST_DIR + "//test"); err == nil {
 		fl.LogAuctionObject(&analytics.AuctionObject{})
+		fl.LogVideoObject(&analytics.VideoObject{})
 		fl.LogAmpObject(&analytics.AmpObject{})
```

Comment:
In order to have a more comprehensive test do you think it won't be much of a hassle to compare the log file at the end of this test? Maybe the the `SameFile(fi1, fi2 FileInfo)` method from the OS package or even asserting against `bytes.Compare` could suffice. What do you think?

Reply:
Comparing log files might be tricky, if logger is including time stamps. you will have to craft some regex's to test the output. Or make your own version of `NewFileLogger()` that creates a non-timestamped logger to generate data.